ztp/source-crs/ImageBasedUpgrade.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: lca.openshift.io/v1alpha1
+kind: ImageBasedUpgrade
+metadata:
+  name: upgrade
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
```

Comment:
Would this CR ever be in a policy that's part of ZTP?

Reply:
Remove the annotation to avoid confusion then

Reply:
Done

Reply:
@jc-rh the [ci job](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-kni_cnf-features-deploy/1727/pull-ci-openshift-kni-cnf-features-deploy-master-ci/1743371685165273088) is failing with the following validation error:
```
Error: missing annotation 'ran.openshift.io/ztp-deploy-wave' in ./source-crs/ImageBasedUpgrade.yaml
```